delete-orphaned-kube-network-load-balancers.sh

```diff
@@ -166,39 +166,69 @@ check_firewalls() {
 
 }
 
+# Check if the target pool has valid member nodes in the current cluster.
+#
+# Returns:
+#   0 - the target-pool has nodes in use in the cluster
+#   1 - the target-pool has no nodes backing it in the cluster
 valid_target_pool() {
     targets=$1
     for i in $targets ; do
```

Comment:
is `i` used in this loop? i only see it in the commented out echo lines.

Reply:
woops that was a rename fail. yea the grep at 183 should be using $i..  good catch.